metashare/api/models.py

```diff
@@ -382,7 +382,7 @@ class Task(
         max_length=100, null=True, blank=True, validators=[validate_unicode_branch],
     )
     commits = JSONField(default=list, blank=True)
-    origin_sha = StringField(null=True, blank=True)
```

Comment:
@wlonk Was this intentional? I thought `null=True` was discouraged for a CharField, but I can revert this if there was a reason to have it.

Reply:
Not intentional that I recall, but the reasons for discouraging `null=True` in character fields are not what they once were; databases handle it much better than they used to, and the only remaining reason is for consistency in your type logic—that is, you usually don't want to have to distinguish between `""` and `None`.